composer.json

```diff
@@ -14,7 +14,8 @@
     "require": {
         "php": "^7.1",
         "ext-pdo": "*",
-        "doctrine/common": "^2.7.1"
+        "doctrine/cache": "^1.0",
+        "doctrine/event-manager": "^1.0@dev"
```

Comment:
Should we release this first?

Reply:
Depends, leaving the decision up to you. 😛 We should definitely have 1.0.0 before DBAL 2.8 either way.

Reply:
Bumped to stable. :+1: